src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
+| Bitcoin             |                 100                 |          38,000x           | [source](https://ccaf.io/cbeci/index/comparisons)                                                                                                 |
+| Youtube             |                 244                 |          94,000 x          | [source](https://thefactsource.com/how-much-electricity-does-youtube-use/)                                                                        |
+| Global data centers |                 200                 |          78,000x           | [source](https://www.iea.org/commentaries/data-centres-and-energy-from-global-headlines-to-local-headaches)                                       |
+| Netflix             |                0.45                 |            175x            | [source](https://s22.q4cdn.com/959853165/files/doc_downloads/2020/02/0220_Netflix_EnvironmentalSocialGovernanceReport_FINAL.pdf)                  |
+| Netflix             |                 94                  |          36,000x           | [source](https://theshiftproject.org/en/article/unsustainable-use-online-video/)                                                                  |
+| Paypal              |                0.26                 |            100x            | [source](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)                       |
+| Gaming in USA       |                 34                  |          13,000x           | [source](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential) |
+| PoW Ethereum        |                 78                  |          30,000x           | [source](https://digiconomist.net/ethereum-energy-consumption)                                                                                    |
+| PoS Ethereum        |               0.0026                |             1x             | [source](https://carbon-ratings.com/eth-report-2022)                                                                                              |
 
-[Digiconomist provides whole-network energy consumption and carbon footprints for Bitcoin and Ethereum](https://digiconomist.net/ethereum-energy-consumption). At the time of writing this article, Bitcoin expends about 200 TWh/yr of energy and emits about 100 MT (megatons) of carbon per year, while generating about 32,000 T of electrical waste from obsolete hardware annually. In comparison, the total energy expenditure for securing Ethereum is closer to **0.01 TWh/yr**.
+Estimates of YouTube's energy expenditure have also been broken down by channel and individual videos. [Those estimates](https://thefactsource.com/how-much-electricity-does-youtube-use/) imply that people consumed >175 times more energy watching Gangnam Style in 2019 alone than the entire Ethereum network uses per year.
```

Comment:
> imply

Should we use stronger language here? i.e. imply > show ? I can see the argument against this, but if we're using these sources IMO we should trust them enough to make more clear-cut statements.

e.g:

```suggestion
Estimates of YouTube's energy expenditure have also been broken down by channel and individual videos. [Those estimates](https://thefactsource.com/how-much-electricity-does-youtube-use/) show YouTube used over 175 times more energy watching Gangnam Style in 2019 than Ethereum uses per year.
```

Reply:
or "demonstrate"

But either way, good with whichever we decide here